README.md

```diff
@@ -1,8 +1,8 @@
 <img src="https://svg.github.io/svgo-logo.svg" width="200" height="200" alt="logo"/>
 
-## SVGO [![NPM version](https://badge.fury.io/js/svgo.svg)](https://npmjs.org/package/svgo)
+## SVGO [![npm version](https://img.shields.io/npm/v/svgo)](https://npmjs.org/package/svgo)
 
-**SVG O**ptimizer is a Nodejs-based tool for optimizing SVG vector graphics files.
+**SVG O**ptimizer is a Node.js-based tool for optimizing SVG vector graphics files.
 ![](https://mc.yandex.ru/watch/18431326)
```

Comment:
Is this still needed? It's blocked here from uBlock origin

Reply:
No idea what this is. You can remove.